pkg/proc/variables.go

```diff
@@ -1244,6 +1247,41 @@ func (v *Variable) maybeDereference() *Variable {
 	}
 }
 
+// loadPtr assumes that v is a pointer and loads its value. v also gets a child
+// variable, representing the pointed-to value. If v is already loaded,
+// loadPtr() is a no-op.
+func (v *Variable) loadPtr() {
+	if v.loaded {
+		return
+	}
+	v.loaded = true
+
+	t := v.RealType.(*godwarf.PtrType)
```

Comment:
This could panic, but I think that's ok as it would signify a programmer error, we shouldn't be calling this on non-pointer types.

Reply:
right; this is called when `v.Kind` is `reflect.Ptr` or `reflect.UnsafePointer`. I expect `v.RealType` to be `*godwarf.PtrType`. That's sane, right?
In my opinion we shouldn't be defensive here, but if you feel otherwise, let me know.

Reply:
Agreed.